rules/models.py

```diff
@@ -0,0 +1,16 @@
+from __future__ import unicode_literals
```

Comment:
imports from __future__ are only relevant for Python 2. we are using Python 3; please remove all from __future__ import statements